apiserver/apiserver/coordinator/coordinator.py

```diff
@@ -240,13 +240,14 @@ def store_game_results(conn, game_output, stats, replay_key, bucket_class,
             sigma=user["sigma"]
         ))
 
-        # Increment number of games played
-        conn.execute(model.bots.update().where(
-            (model.bots.c.user_id == user["user_id"]) &
-            (model.bots.c.id == user["bot_id"])
-        ).values(
-            games_played=model.bots.c.games_played + 1,
-        ))
+        if not challenge:
```

Comment:
I think this would count games with challenge ID 0, but I already used that, so oh well.

Reply:
Yeah, it would and so would the same check at line 144 to not update the ratings. But like you said since challenge 0 is already past it shouldn't matter now.